src/styles/blocks/container.scss

```diff
@@ -0,0 +1,22 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
The same block?

* {
  display: flex;
  flex-wrap: wrap;
  margin: 0 auto;
  box-sizing: border-box;
}

.container {
  width: 1024px;
  padding: 50px 40px;

  &__card {
    margin-right: 48px;
    margin-bottom: 46px;
  }

  &__card:nth-child(4n) {
    margin-right: 0;
  }

  &__card:nth-last-child(-n + 4) {
    margin-bottom: 0;
  }

}

The result on a pic:
![catalog_changes](https://user-images.githubusercontent.com/93187497/183459060-1638bb7e-e5aa-41a8-a900-25dcaffa9227.JPG)

Reply:
Only box-sizing: border-box; was moved to reset.scss